personal-growth/investment-time.html.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Investment Time
+meta_description: >
+  At Nebulab, 10% of the time is dedicated to personal and company projects. Sounds interesting? Read on!
+---
+
+We use 10% of our time for personal growth. During this time, named "Investment Time", we do not work for clients.
+Instead, we focus on initiatives that improve our skills or contribute to the community.
+Here are some ideas for Investment Time:
+
+- Write a post on your personal blog or our [own](working-on-nebulab/blog)
+- Contribute to an open source project
+- Study a new tool or technology
+- Mentor a teammate and teach them a skill
+- Studying a language
+
+This is "Investment Time", because it's time you can invest in yourself to grow as a professional and progress
+in your career. It is the right time to work on personal open source projects (if you have any) or even relaxing
+a little bit if you feel stressed over doing too much client work.
+
+If you come up with an idea and you're not sure whether it's a good candidate for Investment Time, your
+mentor will be able to help you.
+
+## Tracking your Investment Time budget
+
+Every year your Investment Time budget will be 165 hours. You are required to track against this budget by
+using Harvest, there is a task named "Investment Time" under client Nebulab.
+
+Whenever you're doing something for your personal growth, whatever the activity might be, you should be tracking
+against that budget. There are a few exceptions: conferences, All Hands meetings, 1:1s and team retreats are not
+considered Investment Time and should not be tracked. When in doubt, talk to your mentor to figure out if a particular
+activity is Investment Time or something else.
+
+Once you hit the yearly budget you're done for that year and need to wait for the next one to use more Investment
+Time. Considering the full budget is given for every full year, if you joined Nebulab mid-year you might get a
+smaller initial budget.
+
+## Client work and Investment Time
+
+We work for clients from Monday to Friday. It's on you to find the right moment for Investment Time, any arrangement
+works as long as clients are given full priority.
+
+You're free to use your Investment Time whenever you like, there are a few options that are known to work:
+
+- first/last hour of the work day
+- a few hours when the client's team workday hasn't started yet
+- when client work slows down, reserve a few days to focus on Investment Time
```

Comment:
In order to keep the list consistent, I'd use uppercase and `.` at the end of each item.